packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -469,3 +566,10 @@ export const ALL_STORES = [
   DbTargetGlobal.store,
   DbTargetDocument.store
 ];
+
+/**
+ * The list of all default IndexedDB stores used throughout the SDK. This is
+ * used when creating transactions so that access across all stores is done
+ * atomically.
+ */
+export const ALL_STORES = [...V1_STORES];
```

Comment:
This is weird...  I wonder if we should get rid of the V1_STORES  list or else do:

```
// No new stores added in V2.
const V2_STORES = V1_STORES;

...
export const ALL_STORES = V2_STORES;
```
(maybe V2_STORES is overkill, but I think we should at least have the comment explaining why we're using V1_STORES even though SCHEMA_VERSION is 2)

Reply:
I got rid of `V1_STORES`, since there are not [yet] any new stores. Whoever adds new stores can decide how to split them up. Probably not with `V2_STORES`, since that won't be v2.